frontend/apps/reader/modules/readergesture.lua

```diff
@@ -953,6 +953,9 @@ function ReaderGesture:setupGesture(ges, action)
             "paging_swipe",
             "rolling_swipe",
         }
+        overrides_horizontal_edge = {
+            "swipe_link"
+        }
```

Comment:
Shouldn't it also overrides the ones that `overrides_vertical_edge` does?
And both `overrides_vertical_edge`  and `overrides_horizontal_edge` might also need to override `readerconfigmenu_swipe` and `readermenu_swipe` (they may not overlap as DTAP_ZONE_MENU.x=1.8 in defaults.lua, but some people may increase that).

For info, some months ago (so it may no more be correct), I checked the various gestures overrides and possible conflicts in https://github.com/koreader/koreader/issues/4856#issuecomment-478274191.

Reply:
Fully agree :)